README.md

```diff
@@ -29,12 +22,30 @@ Open terminal and go to root directory of the project and run:
 pytest
 ````
 
+## Configuration
+
+The properties in data_service/config/.env are meant for Bucket configuration 
+and are applicable when the service is deployed within a Docker container.
+
+## Running application from command line
+```
+% pwd
+/microdata-data-service
+
+% ./start_local.sh 
+```
+Pay attention to environment variables set in start_local.sh
+
 ## Running/debugging application in IntelliJ IDEA
 Go to Edit configurations... -> Add New Configuration -> Python.
 
 Set "Script path" to `[your-path]/application.py`
 
 Set "Working directory" to `[your-path]/microdata-data-service`
 
+Set environment variables as in start_local.sh
```

Comment:
Add a note to check .env file. Variables set in start_local.sh will **overwrite** those from .env file.